api/RTCRtpSender.json

```diff
@@ -355,7 +355,7 @@
               "version_added": "75"
             },
             "edge": {
-              "version_added": "12",
+              "version_added": "13",
               "version_removed": "79"
```

Comment:
This `version_removed` has to be wrong, can you double check?

Reply:
Looks like these aren't covered by the collector, but they are indeed wrong.  Fixing!